### ISSUE TITLE
403 Forbidden

### ISSUE THREAD

**Issue by samimaken** [unicodeveloper/laravel-paystack#176]
Hi, 
I am getting this error

 Client error: `POST https://api.paystack.co/transaction/initialize` resulted in a `403 Forbidden` response:
    <!DOCTYPE html><html lang="en-US"><head><title>Just a moment...</title><meta http-equiv="Content-Type" content="text/htm (truncated...)

followed all steps,
added keys in the .env file

` public function accept(Request $request)
    {

        try {
            return Paystack::getAuthorizationUrl()->redirectNow();
        } catch (\Exception $e) {
            dd($e);
            return Redirect::back()->with('error', 'The paystack token has expired. Please refresh the page and try again.');
        }
    }`

**Comment by Hateem76** [comment]
@samimaken did you solve it?

**Comment by samimaken** [comment]
Yes,  you have to use a Nigerian or South African VPN to use this API.

**Comment by Hateem76** [comment]
> Yes, you have to use a Nigerian or South African VPN to use this API.

Ohkay, thanks mate, for saving my day...
